### PR TITLE
Add modal to create new project

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-571

#### Link to staging deployment URL 

https://ui-agi-ui-133.scp-staging.biomage.net/

#### Links to any Pull Requests related to this

N.A.

#### Anything else the reviewers should know about the changes here

- Changed the link for Data Upload to `/data-management`
- Added project name validation. If project name is invalid, a popup comes up saying name is invalid
- Disable navigation links if there no `experiment` chosen

# Changes
### Code changes

- Moved `pages/experiments/[experimentId]/data-upload` folder to `/pages/data-management`
- Added new component to get modal
- Fixed `Header` component so it can work without 

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [X] Unit tests written
- [X] Tested locally with Inframock
- [X] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by vickymorrison** [comment]
A few comments:
1. Cellscope should not have a capital S
2. The user should not be able to create a project with no name (i.e. if the user doesn't type anything but clicks 'Create Project' then a message should appear to state that this is a required field and they cannot continue)
3. When the user is on the data management home screen the navigation on the left should show the user (in light blue) where they are. Currently, the data management module does not show the user where they are (see screenshot 1) - it should be like in data exploration (screenshot 2).
Screenshot 2021-03-01 at 13 06 19
Screenshot 2021-03-01 at 13 15 58

File: src/__test__/components/ContentWrapper.test.jsx [hunk 1]
```diff
@@ -10,11 +10,15 @@ import ContentWrapper from '../../components/ContentWrapper';
...
-  })),
+  useRouter: jest.fn()
+    .mockImplementationOnce(() => ({ // 1st test
+      query: {
+        experimentId: '1234',
+      },
+    }))
+    .mockImplementationOnce(() => ({ // 2nd test
+      query: {},
+    })),
```
Comment (aerlaut) [inline src/__test__/components/ContentWrapper.test.jsx:21]: This is done to simulate going to `/data-management`, where `experimentId` is undefined

**Review by vickymorrison — APPROVED** [review]

**Comment by aerlaut** [comment]
Thanks for the approve @vickymorrison ! Waiting for someone to review engineering-wise

File: src/__test__/components/ContentWrapper.test.jsx (hunk 1 above)
Comment (ivababukova) [reply src/__test__/components/ContentWrapper.test.jsx:21]: good idea!

**Review by marcellp — APPROVED** [review]

**Pull request merged**